corehq/apps/app_manager/templates/app_manager/partials/mobile_report_configs.html

```diff
@@ -69,14 +69,20 @@
                             <select class="form-control"
                                     data-bind="options: availableReportIds, optionsText: $root.defaultReportTitle, value: reportId">
                             </select>
-                            <label>{% trans "Report UUID to use in custom expressions:" %}</label>
-                            <span data-bind="text: uuid"></span>
+                            <div data-bind="css: { 'has-error': $root.validateSlug(instanceId()) }">
+                                <label>{% trans "Report code to use in custom expressions:" %}</label>
+                                <span data-bind="visible: !$root.supportCustomUcrSlug, text: uuid"></span>
+                                <input type="text" class="form-inline"
```

Comment:
Noticed while looking on staging - `form-inline` isn't doing much of anything here. If you use `form-control` on this input and then `form-inline` on the parent div, you'll get the round corners, etc. on the input and the layout should look reasonable.

Reply:
Nice, thanks, just updated it.